src/audio/code_8019AF00.c

```diff
@@ -330,7 +330,7 @@ u8 sSeqFlags[] = {
     SEQ_FLAG_ENEMY,                    // NA_BGM_CAVERN
     SEQ_FLAG_RESUME | SEQ_FLAG_ENEMY,  // NA_BGM_MILK_BAR
     SEQ_FLAG_FANFARE,                  // NA_BGM_ZELDA_APPEAR
-    0,                                 // NA_BGM_SARIAS_SONG
+    0,                                 // NA_BGM_WOODS_OF_MYSTERY
```

Comment:
This rename is tricky because I've thought of naming it this before, as well as `NA_BGM_LOST_WOODS` but it is just an identical track to OoT's saria's song, so it feels weird to give it a different name. You can even see it in the name `MAYOIMORI`.

The reason I went with `SARIAS_SONG` over `LOST_WOODS` was that `LOST_WOODS` is a different region in MM so using that name here would be confusing, even though the OG name `MAYOIMORI` translates to lost woods.

Note that there's a similar issue with `NA_BGM_ROMANI_RANCH` which is really just `NA_BGM_LON_LON` (`RONRON`) so there's already a degree of inconsistency with this

Reply:
Yes but `Woods of mystery` is not `Lost woods`, the actual place this music plays, the maze in southern swamp where we go chase the monkey to find the witch, is called Woods of mystery, so it kinda makes sense to name it after the area instead of the non-existent character.
(although either is fine with me)

Reply:
Yeah sorry, that was the exact point I was trying to make, although maybe I wasn't clear.

OoT Lost Woods -> This music plays
MM Lost Woods -> This music does not play (pre-clocktown location)
MM Woods of Mystery -> This music play (monkey puzzle)

The original name, `MAYOIMORI`, translates to `LOST WOODS`, but that name would be confusing in MM because this track does not play in MM's Lost Woods, it instead plays in the Woods of Mystery.

In trying to preserve the track name between the two games (because it is an identical track between the 2 games), I went with Saria's Song because it does match the actual ocarina Saria's Song exactly.

Plus, this song isn't only used in the Woods of Mystery. This is also the track used by the scarecrow when dancing and skipping ahead to the next half day

Reply:
Changed it back. Still not the biggest fan, but this is clearly a fight I'm not winning.